### PR TITLE
Whats new 4 may

### PR DESCRIPTION
**Pull request by molebox**
This PR adds a link to the 2.22.0 release notes.

It also refactors how we display this info and adds a highlights and additions sections

### CODE REVIEW

File: content/600-about/00-whats-new-in-prisma-docs.mdx [hunk 1]
```diff
@@ -12,6 +12,28 @@ Full release notes for Prisma itself, including breaking changes, are published
 
 </TopBlock>
 
+## May 04 2021
+
+That we refactored and expanded the deployment section, refactored connection management
```
Comment (mhwelander) [inline content/600-about/00-whats-new-in-prisma-docs.mdx]: Remove?

File: content/600-about/00-whats-new-in-prisma-docs.mdx [hunk 2]
```diff
@@ -12,6 +12,28 @@ Full release notes for Prisma itself, including breaking changes, are published
...
+### Release 🚀
+
+`prisma db push` is now out of preview!
+
+- [2.22.0](https://github.com/prisma/prisma/releases/tag/2.22.0) documentation
+
+### New documentation
+
+#### Highlights 💡
+
```
Comment (mhwelander) [inline content/600-about/00-whats-new-in-prisma-docs.mdx:35]: Would move this + User/Post stuff to 'Other documentation' or 'other highlights'

File: content/600-about/00-whats-new-in-prisma-docs.mdx [hunk 3]
```diff
@@ -12,6 +12,28 @@ Full release notes for Prisma itself, including breaking changes, are published
...
+The past month we refactored and expanded the [deployment](../guides/deployment/deployment) and [connection management](../guides/performance-and-optimization/connection-management) sections.
+
+#### Additions:
+
+- In depth [explanation](./about-the-docs#the-user-and-post-data-model) about our choice to use the `User` and `Post` data models across the docs
+- [`db push`](../concepts/components/prisma-migrate/db-push) no longer in preview!
+- Multiple provider support [no longer supported](../concepts/components/prisma-schema/data-sources)
+
```
Comment (mhwelander) [inline content/600-about/00-whats-new-in-prisma-docs.mdx:41]: We can add the video now: https://www.youtube.com/watch?v=zBIQUdqgfJM

File: content/600-about/00-whats-new-in-prisma-docs.mdx (hunk 1 above)
Comment (molebox) [reply content/600-about/00-whats-new-in-prisma-docs.mdx]: oops!

File: content/600-about/00-whats-new-in-prisma-docs.mdx [hunk 4]
```diff
@@ -12,6 +12,28 @@ Full release notes for Prisma itself, including breaking changes, are published
 
 </TopBlock>
 
+## May 04 2021
+
+That we refactored and expanded the deployment section, refactored connection management
+
+### Release 🚀
+
+`prisma db push` is now out of preview!
```
[the remark this answers is not in the document]
Comment (molebox) [reply content/600-about/00-whats-new-in-prisma-docs.mdx:19]: Good shout, have made changes

**Comment by molebox** [comment]
@mhwelander done, done and done :-)

**Pull request merged**